_maps_2024/15-luming.md

```diff
@@ -0,0 +1,7 @@
+---
+day: 15
+creator: Luming Xu
+title: Potential Innovation Districts in Shenzhen, China
+thumbnail: /assets/thumbnails/2024/15-Luming-My Data.png
```

Comment:
**suggestion (non-blocking):** @Bluebird-hi I _think_ this file name will be fine, but spaces in file names can sometimes lead to unexpected results. I recommend to replace the space with a dash in the file name and resubmit.

Reply:
okay! I’ll update later!